ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+import pytest
+import pathlib
+import textwrap
+
+logger = logging.getLogger("testrunner")
+
+CONTENT = """---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: leap
+spec:
+    containers:
+    - name: app
+      image: opensuse/leap:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle12sp4
+spec:
+    containers:
+    - name: app
+      image: registry.suse.com/suse/sles12sp4:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle15
+spec:
+    containers:
+    - name: app
+      image: registry.suse.com/suse/sle15:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle15sp1
+spec:
+    containers:
+    - name: app
+      image: registry.suse.de/suse/containers/sle-server/15/containers/suse/sle15:15.1
+      command: ['/bin/sh', '-c', 'sleep 3600']"""
+
+
+@pytest.fixture
+def setup_manifest():
+    cwd = pathlib.Path.cwd()
+    p = cwd / "pods.yaml"
```

Comment:
I just want to note that this is simultaneously the ugliest and most creative overloading of the `/` operator I've seen. :D